### PR TITLE
#2454 Multiselect context menu for PI 

### LINKED ISSUE #2454: Multiselect context menu for PI
same as for MoBi (https://github.com/Open-Systems-Pharmacology/MoBi/issues/508)

### PR DESCRIPTION
**Pull request by peterdew**
Fixes #2454

# Description

A multi-select contextmenu is added for parameter identifications with the options to delete and to add to journal.

## Type of change

Please mark relevant options with an `x` in the brackets.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires documentation changes (link at least one [user](https://github.com/Open-Systems-Pharmacology/docs) or [developer](https://github.com/Open-Systems-Pharmacology/developer-docs) documentation issue):
- [ ] Algorithm update - updates algorithm documentation/questions/answers etc.
- [ ] Other (please describe):

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Integration tests
- [ ] Unit tests
- [x] Manual tests 
- [ ] No tests required

# Reviewer checklist

Mark everything that needs to be checked before merging the PR.

- [ ] Check if the code is well documented
- [ ] Check if the behavior is what is expected
- [ ] Check if the code is well tested
- [ ] Check if the code is readable and well formatted
- [ ] Additional checks (document below if any)
- [ ] Check if documentation update issue(s) are created if the option `This change requires a documentation update` above is selected

# Screenshots (if appropriate):
image

# Questions (if appropriate):

### CODE REVIEW

**Review by rwmcintosh — APPROVED** [review]

**Comment by rwmcintosh** [comment]
Somehow this did not get submitted along with the MoBi one. I updated it.

File: src/PKSim.Presentation/Presenters/ContextMenus/MultipleSensitivityAnalysesContextMenu.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      private readonly IContainer _container;
+
+      public MultipleSensitivityAnalysesContextMenuFactory(IContainer container)
+      {
+         _container = container;
+      }
+
+      protected override IContextMenu CreateFor(IReadOnlyList<ClassifiableSensitivityAnalysis> nodes, IPresenterWithContextMenu<IReadOnlyList<ITreeNode>> presenter)
+      {
+         var sensitivityAnalysisList = nodes.Select(pi => pi.SensitivityAnalysis).ToList().AsReadOnly();
```
Comment (msevestre) [inline src/PKSim.Presentation/Presenters/ContextMenus/MultipleSensitivityAnalysesContextMenu.cs:28]: AsReadOnly? What is this?

Comment (msevestre) [reply src/PKSim.Presentation/Presenters/ContextMenus/MultipleSensitivityAnalysesContextMenu.cs:28]: List implements IREadOnlyList anyways. so why do we need the two? 🤔

**Review by msevestre — APPROVED** [review]
just one question. Rest looks fine

File: src/PKSim.Presentation/Presenters/ContextMenus/MultipleSensitivityAnalysesContextMenu.cs (hunk 1 above)
Comment (rwmcintosh) [reply src/PKSim.Presentation/Presenters/ContextMenus/MultipleSensitivityAnalysesContextMenu.cs:28]: Removed.

**Pull request merged**